00_Base/src/util/util.ts

```diff
@@ -14,9 +14,13 @@ export enum CountryCode {
   MX = 'MX',
 }
 
-export const plainToClass = <T>(constructor: Constructable<T>, plain: T): T =>
+export const plainToClass = <T>(
+  constructor: Constructable<T>,
+  plain: T,
+  excludeExtraneousValues = true,
+): T =>
   plainToInstance(constructor, plain as T, {
-    excludeExtraneousValues: true,
+    excludeExtraneousValues,
```

Comment:
Why this functional change?

Reply:
this helper method was hardly used, and it was previously always setting `excludeExtraneousValues` to true, but that will not parse child classes so I adjusted this to be a variable so we can default to true but are able to pass in `false` where needed